lib/hooks/create/unique.js

```diff
@@ -23,7 +23,9 @@ module.exports = settings => {
       'establishment.update-conditions',
       'establishment.update-billing',
       'profile.resend-email',
-      'profile.confirm-email'
+      'profile.confirm-email',
+      'pil|project|establishment.suspend',
+      'pil|project|establishment.reinstate'
```

Comment:
Are you _sure_ this won't match all pil and project tasks? I'd be tempted to replace with `*.suspend` and `*.reinstate` for a blend of simplicity and a certain answer to that question?

Reply:
Having tried this out a little using minimatch in CLI it seems to not match at all.

```
> match('pil.suspend', 'pil|project|establishment.suspend')
false
> match('pil.suspend', '+(pil|project|establishment).suspend')
true
```

It looks like the syntax needs to be `+(a|b|c).action`.